### ISSUE TITLE
[BUG] - issue::checkpoints are not saving the information and weights after few days. givinig different results.

### ISSUE THREAD

**Issue by anushka192001** [pytorch/tutorials#2856]
### Add Link

when loading a saved pytorch model after few days giving very bad results.
hamming score suddenly dropped from 75% to 0.1% and flat score dropped from 65% to 0.3%.

torch.save(model, 'models/model_0.pth')
model = torch.load('models/model_0.pth')

checkpoints are not saving the information and weights after few days.

### Describe the bug

when loading a saved pytorch model after few days giving very bad results.
hamming score suddenly dropped from 75% to 0.1% and flat score dropped from 65% to 0.3%.

torch.save(model, 'models/model_0.pth')
model = torch.load('models/model_0.pth')

checkpoints are not saving the information and weights after few days.

### Describe your environment

visual studio 

cc @albanD

**Comment by svekars** [comment]
thank you for filing this issue. Can you please:
- Add a link to the tutorial this issue is about?
- Describe your environment in more details - which version of PyTorch, CUDA, Python. Did you run as a notebook or as a script.

**Comment by anushka192001** [comment]
i am talking about saving nd loading the model in pytorch https://pytorch.org/tutorials/beginner/saving_loading_models.html
i am using PyTorch version 1.11.0 with CUDA 11.5  and Python version: 3.10.12  in visual studio in my jupyter notebook.
i am also using 2 nvidia gpu's in it 
GPU 0: Tesla T4, CUDA Capability 7.5
GPU 1: Tesla T4, CUDA Capability 7.5

when i save the distllbert trained model and then loading it then showing different results accuracy dropped from 75% to 0.1 %

torch.save(model, 'models/model_0.pth')
model = torch.load('models/model_0.pth')

**Comment by anushka192001** [comment]
@svekars can you share you mail id so that i can directly share the code file with you.

**Comment by svekars** [comment]
cc: @MatthewInkawhich

**Comment by malfet** [comment]
> @svekars can you share you mail id so that i can directly share the code file with you.

@anushka192001 this is an open source project, so all the code snippets discussed here should be public.

**Comment by anushka192001** [comment]
data = data[['concatenated_sentence','Final_Functional_Level']] 
data.head() 


	text	labels
15523	retired marketing communications executive vic...	[0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...
8324	securities trader elite client services financ...	[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, ...
4785	communications director disability business re...	[0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, ...
14486	marketing production senior marketing communic...	[0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...
14784	upgrade bcr internet banking click banking geo...	[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, ...



print(len(data)) 
new_df = data 


# Sections of config

# Defining some key variables that will be used later on in the training
MAX_LEN = 512
TRAIN_BATCH_SIZE = 32
VALID_BATCH_SIZE = 32
EPOCHS = 15
LEARNING_RATE = 1e-05
tokenizer = DistilBertTokenizer.from_pretrained('distilbert-base-uncased', truncation=True, do_lower_case=True)

**Comment by malfet** [comment]
This does not seem to be a tutorials problem, if you believe `torch.load` corrupts data, please file an issue against https://github.com/pytorch/pytorch/issues but do include some sort of a reproducible example (code you've pasted above is not a fully functional code, but snippets from it seems to work. 

So closing one against tutorials, but please do not hesitate to open one against PyTorch, but do provide enough information so that it could be reproduced by another party.
